dm/task-configuration-file-full.md

```diff
@@ -140,6 +140,16 @@ syncers:
     # `DELETE FROM tb WHERE a=1; DELETE FROM tb WHERE a=2` will become `DELETE FROM tb WHERE (a) IN (1),(2)`, where "a" is the primary key
     multiple-rows: true
 
+# Configuration arguments of continuous data validation (validator).
+validators:
+  global:                # Configuration name.
+    # full: validates the data in each row is correct.
+    # fast: validates whether the row is successfully migrated to the downstream.
+    # none: does not validate the data.
+    mode: full           # Possible values are "full", "fast", and "none". The default value is "none", which does not validate the data.
+    worker-count: 4      # The number of validation workers in the background. The default value is 4.
+    row-error-delay: 30m # If a row is not validated within the specified time, it will be marked as an error row. The default value is 30m, which means 30 minutes.
```

Comment:
ditto

Reply:
```suggestion
    row-error-delay: 30m # If a row cannot pass the validation within the specified time, it will be marked as an error row. The default value is 30m, which means 30 minutes.
```